contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -143,6 +155,17 @@ def _transmit(self, envelopes):
                 data = json.loads(text)
             except Exception:
                 pass
+
+        if self._is_stats_exporter() and \
+            not state.get_statsbeat_shutdown() and \
```

Comment:
I can see you check if shutdown was called in several places, is the exporter process for Statsbeat expected to keep running after shutdown?

Reply:
This is for very specific race conditions in which multiple threads could be accessing the same piece of "check if we need to shutdown" logic. It also serves as a good sanity check to prevent from any statsbeat logic from executing if the statsbeat exporte ris already shutdown.